### PR TITLE
Add reference to the Azure Fluid Relay

### PR DESCRIPTION
**Pull request by szilvaa**

### CODE REVIEW

File: docs/content/start/faq.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ Microsoft has developed an M365-specific Fluid service designed to enable soluti
 ecosystem. There will be ways for Fluid Framework developers to operate in M365 but those integration points are
 not available yet.
 
-Microsoft is also looking at providing a more general Fluid service but no specifics are available at this time.
+Microsoft has also [announced](https://developer.microsoft.com/en-us/office/blogs/whats-new-in-microsoft-365-platform-at-build-2021/) Azure Fluid Relay, a production fluid service, at Build 2021.
```
Comment (tylerbutler) [inline docs/content/start/faq.md:159]:
```suggestion
Microsoft has also [announced](https://developer.microsoft.com/en-us/office/blogs/whats-new-in-microsoft-365-platform-at-build-2021/) Azure Fluid Relay, a fully managed Fluid service, at Build 2021.
```

**Comment by tylerbutler** [comment]
@szilvaa Thanks for the contribution! I had some small suggestions. @ahmedbisht Is there any other info you'd like to include or changes you'd make?

**Comment by ahmedbisht** [comment]
Looks good! Thank you for the contribution, @szilvaa.
The service is still in Private Preview as the //build announcement mentions. Once we get closer to Public Preview, we can update this reference.

**Review by tylerbutler — APPROVED** [review]

**Comment by ahmedbisht** [comment]
@tylerbutler , Any updates needed for this or any other place on fluidframework.com now that we are GA?

**Pull request merged**